src/solvers/dgmulti/flux_differencing_gsbp.jl

```diff
@@ -125,9 +123,9 @@ function entropy_projection!(cache, u, mesh::VertexMappedMesh, equations, dg::DG
 
   # transform entropy to conservative variables on face values
   entropy_projected_face_values = view(entropy_projected_u_values, face_indices, :)
-  @threaded for i in eachindex(entropy_var_face_values)
-    entropy_projected_face_values[i] = entropy2cons(entropy_var_face_values[i], equations)
-  end
+  entropy2cons!(entropy_projected_face_values, entropy_var_face_values, equations)
```

Comment:
Interesting - a `view` of a `StructArray` creates another `StructArray`, so this will dispatch to the specialization with `StructArray` arguments. 

This isn't a suggestion, just a comment to remind myself of why this works.

Reply:
Yes, creating a `view` of a `StructArray` creates `view`s of the underlying memory and wraps a `StructArray` around it.